force-app/main/releaseGating/lwc/releaseGateItem/releaseGateItem.js

```diff
@@ -21,24 +35,24 @@ export default class ReleaseGateItem extends LightningElement {
     };
 
     get gateActive() {
-        return this.gate.status === "active";
+        return this.gateStatus === "active";
```

Comment:
totally nitpick, but I was wondering if it makes sense to move these flags as constants (to the top of the class)

Reply:
Totally makes sense, replaced with constants